### PR TITLE
test PR

### PR DESCRIPTION
**Pull request by muodov**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/muodov/kociemba/pull/24" target="_blank">ReviewNinja

File: kociemba/search.py [hunk 1]
```diff
@@ -195,9 +195,9 @@ def totalDepth(self, depthPhase1, maxDepth):
         maxDepthPhase2 = min(10, maxDepth - depthPhase1)    # Allow only max 10 moves in phase2
         for i in xrange(depthPhase1):
```
Comment (muodov) [inline kociemba/search.py]: hey fix this! !fix

File: kociemba/search.py [hunk 2]
```diff
@@ -157,9 +157,9 @@ def solution(self, facelets, maxDepth, timeOut, useSeparator):
             # +++++++++++++ compute new coordinates and new minDistPhase1 ++++++++++
             # if minDistPhase1 =0, the H subgroup is reached
             mv = 3 * self.ax[n] + self.po[n] - 1
-            self.flip[n + 1] = CoordCube.flipMove[self.flip[n]][mv]
-            self.twist[n + 1] = CoordCube.twistMove[self.twist[n]][mv]
-            self.slice[n + 1] = CoordCube.FRtoBR_Move[self.slice[n] * 24][mv] / 24
+            self.flip[n + 1] = CoordCube.flipMove[self.flip[n], mv]
+            self.twist[n + 1] = CoordCube.twistMove[self.twist[n], mv]
+            self.slice[n + 1] = CoordCube.FRtoBR_Move[self.slice[n] * 24, mv] / 24
             self.minDistPhase1[n + 1] = max(
```
Comment (muodov) [inline kociemba/search.py]: this is nice

File: kociemba/search.py (hunk 1 above)
Comment (muodov) [inline kociemba/search.py]: fixed! !fixed

Comment (muodov) [inline kociemba/search.py]: !resolve

File: kociemba/coordcube.py [hunk 3]
```diff
@@ -31,13 +31,15 @@ def load_cachetable(name):
     try:
         with open(os.path.join(cache_dir, name + '.pkl')) as f:
             obj = cPickle.load(f)
```
Comment (muodov) [inline kociemba/coordcube.py]: NOO!!

**Comment by muodov** [comment]
Review status: 0 of 26 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[kociemba/search.py, line 194 \[r1\]](https://reviewable.io:443/reviews/muodov/kociemba/24#-KM-HBNvFu-chIAkazYG:-KM-HBNvFu-chIAkazYH:160408567) ([raw file](https://github.com/muodov/kociemba/blob/e692077fcff8f8ba7af63425d09ba8f191e48d68/kociemba/search.py#L194)):*
> ```Python
>         mv = 0
>         d1 = 0
>         d2 = 0
> ```

hey

---

*[kociemba/search.py, line 196 \[r1\]](https://reviewable.io:443/reviews/muodov/kociemba/24#-KM-Gu9-eSjQnOMScy0A-r1-196:-KM-H1T3_teSuxUwUyOQ:-1965200042) ([raw file](https://github.com/muodov/kociemba/blob/e692077fcff8f8ba7af63425d09ba8f191e48d68/kociemba/search.py#L196)):*
<details><summary><i>Previously, muodov (Maxim Tsoy) wrote…</i></summary>
> !resolve
</details>
esefsdfsdf

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/muodov/kociemba/24)*

**Pull request closed without merging**